mlir/lib/Bindings/Python/IRAttributes.cpp

```diff
@@ -694,6 +694,8 @@ class PyDenseElementsAttribute
         // f16
         assert(view.itemsize == 2 && "mismatched array itemsize");
         bulkLoadElementType = mlirF16TypeGet(context);
+      // } else if (format == "?") {
```

Comment:
Did you possibly push an incorrect patch? This appears to just be commented out

Reply:
Woops, yes. For this to work we technically need an upstream LLVM change to land. I was uncommenting to test.